daprdocs/content/en/concepts/dapr-services.md

```diff
@@ -0,0 +1,45 @@
+---
+type: docs
+title: "Dapr Services"
+linkTitle: "Services"
+weight: 100
+---
+
+
+Dapr supports multiple services that help sidecar - sidecar, sidecar - app and sidecar - system communication. Following are the services and the way to leverage them with your application.
```

Comment:
This sentence is not clear and seems out of place. daprd is the sidecar executable, how does this page relate to sidecar to sidecar or sidecar to app communication. 
Also, what services are described below?